gnuplot.el

```diff
@@ -2,8 +2,8 @@
 
 ;; Copyright (C) 1998, 2011 Phil Type and Bruce Ravel, 1999-2012 Bruce Ravel
 
-;; Author:     Bruce Ravel <bruceravel1@gmail.com> and Phil Type
-;; Maintainer: Bruce Ravel <bruceravel1@gmail.com>
+;; Authors:    Jon Oddie, Bruce Ravel, Phil Type, and others
```

Comment:
The function `lm-authors` (from the built-in `lisp-mnt.el` library) has to be able to extract this, and it is not a very good function. (I'll should finally get around to contributing a better implementation.) As a result you have to spell it like this (multiple lines, singular "Author"):

```emacs-lisp
;; Author: Jon Oddie
;;   Bruce Ravel
;;   Phil Type
;;   and others
```

I would omit the `and others`.